KnockKnock-iOS/Scenes/Login/LoginInteractor.swift

```diff
@@ -23,54 +27,74 @@ final class LoginInteractor: LoginInteractorProtocol {
   var worker: LoginWorkerProtocol?
   var router: LoginRouterProtocol?
 
-  func fetchLoginResult(
-    source: LoginViewProtocol,
-    socialType: SocialType
-  ) {
+  func fetchLoginResult(socialType: SocialType) {
+
     self.worker?.fetchLoginResult(
+      appleLoginResultDelegate: self,
       socialType: socialType,
-      completionHandler: { loginResponse, loginInfo in
-
-        // 회원 판별
-        // 회원 o -> 토큰 저장 후 홈 화면 진입 / 회원 x -> 프로필 설정화면(회원가입)
-        if loginResponse.isExistUser {
-          self.saveTokens(loginResponse: loginResponse)
-          self.popLoginView(source: source)
-          NotificationCenter.default.post(name: .loginCompleted, object: nil)
-        } else {
-          self.navigateToProfileSettingView(
-            source: source,
-            loginInfo: loginInfo
-          )
-        }
+      completionHandler: { response, loginInfo in
+
+        self.isExistUser(
+          response: response,
+          loginInfo: loginInfo
+        )
       }
     )
   }
+  
+  /// 회원 판별
+  /// isExisted: 기존 회원 여부
+  /// 회원 o -> 토큰 저장 후 홈 화면 진입 / 회원 x -> 프로필 설정화면(회원가입)
+  func isExistUser(response: AccountResponse, loginInfo: LoginInfo) {
```

Comment:
```suggestion
  func checkExistUser(response: AccountResponse, loginInfo: LoginInfo) {
```
요런 네이밍이 좋지 않을까요? boolean 값을 리턴하는 함수 또는 프로퍼티가 아니니까요 ㅎ